test/addons/rook-cephfs/start

```diff
@@ -9,43 +9,57 @@ import sys
 import drenv
 from drenv import kubectl
 
-FILE_SYSTEMS = ["test-fs1", "test-fs2"]
 STORAGE_CLASS_NAME_PREFIX = "rook-cephfs-"
+FS_NAMES = ["test-fs1", "test-fs2"]
+CEPHFS_VOLUME_SNAPSHOT_CLASSES = [
+    "csi-cephfsplugin-snapclass",
+    "csi-cephfsplugin-snapclass-2",
+]
+
+STORAGE_CLASSES_MAP = [
+    {"fs": fs, "name": f"{STORAGE_CLASS_NAME_PREFIX}{fs}", "vsc": vsc}
+    for fs, vsc in zip(FS_NAMES, CEPHFS_VOLUME_SNAPSHOT_CLASSES)
+]
```

Comment:
This is not a map but a list. Should be the same static list with all the info as in rbd-mirror/start.